cmd/clusterctl/examples/openstack/machine-deployment.yaml.template

```diff
@@ -22,7 +22,6 @@ spec:
           kind: "OpenstackProviderSpec"
           flavor: m1.medium
           image: <Image Name>
-          sshUserName: <SSH Username>
```

Comment:
I would keep those for debug purpose.... or at least show a way to user how to debug issues

and sshUsername and keyname below should be a group... remove or keep them both

Reply:
I'm not sure. 

keyname is used to inject the given keypair. So that's absolutely necessary to jump on the host later, without it, it won't work. sshUserName would be only to document to the user inside the Machine resource what the name of his os user is. I'm not sure if he already knows that and if it's strictly necessary to put this inside the Machine resource

Reply:
But I also don't have very strong opinions on keeping or deleting them both. I think deleting both is not really an option because without keyname we don't inject the key which makes it possible in the first place to ssh onto the host.

I guess I would go with documenting how to ssh onto the host.